src/pages/en/core-concepts/astro-pages.md

```diff
@@ -71,14 +71,28 @@ This is my page, written in **Markdown.**
 
 > ⚠️ This feature is currently only supported with the `--experimental-static-build` CLI flag. This feature may be refined over the next few weeks/months as SSR support is finalized.
 
-Non-HTML pages, like `.json` or `.xml`, can be built from `.js` and `.ts`. 
+Non-HTML pages, like `.json` or `.xml`, or even non-text assets like images can be built using **File Routes**. **File Routes** must end with the `.js` or `.ts` extension.
```

Comment:
```suggestion
Non-HTML pages, like `.json` or `.xml`, or even non-text assets like images can be built using **File Routes**. 

**File Routes** must end with the `.js` or `.ts` extension and the source file must exist within the `src/pages/` directory.
```
(Does something like this address @delucis ' comment? Is this clear enough to indicated that it can be in any subdirectory of /pages?)

Reply:
Yes, this sounds really good 👍